docs/api/combineReducers.md

```diff
@@ -7,6 +7,8 @@ reducing function you can pass to [`createStore`](createStore.md).
 
 The resulting reducer calls every child reducer, and gather their results into a single state object. The shape of the state object matches the keys of the passed `reducers`.
 
+Reducer function names are important. Don't be confused by the fact that reducer parameter is named `state`. *Single reducer in the combination does not update compete state of the store.* Its name is the key of the value in the store object it gets and updates. For example, reducer named `todos` receives and changes only `state.todos`.
```

Comment:
I'd say it's not exactly function names but keys in object you pass to combineReducers. They match function names when you use export syntax.